processor/metric/system/_meta/fields.yml

```diff
@@ -0,0 +1,80 @@
+- key: system
```

Comment:
How did you come up with this?
I checked it with the `metricbeat` example, but that has `metrics.samples.system` and `metrics.samples.memory` and not `system.memory`.

Reply:
This is from https://www.elastic.co/guide/en/beats/metricbeat/current/exported-fields-system.html#_memory_fields_5 - does that look right?

Reply:
So some metrics have a defined type and format, and all the others are also indexed, but dependent on the first value, right?

Reply:
exactly